test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BasicAuthTests.cs

```diff
@@ -208,5 +211,50 @@ public async Task GivenAUserWithExportPermissions_WhenExportResources_TheServerS
             Uri contentLocation = await tempClient.ExportAsync();
             await tempClient.CancelExport(contentLocation);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenAUserWithNoDataConvertPermissions_WhenDataConvert_TheServerShouldReturnForbidden()
+        {
+            if (!_isUsingInProcTestServer)
```

Comment:
>_isUsingInProcTestServer [](start = 17, length = 24)

Can you add comment what is difference between Export and DataConvert action?
Export doesn't check for InProcTestServer, why DataConvert need to do that?

Reply:
Actually there's not difference. I will remove this line. (At first this condition was added for PR/CI checks as the deployed environment does not enable data convert.